### ISSUE TITLE
Clicked element stays in active state

### ISSUE THREAD

**Issue by teknetia** [chaptergy/noctis-grey#2]
When I click on an element it stays in an active state until I click on something outside of the active area (the header, background, etc.) like in the images attached. I can't replicate it in other themes and also can't figure out exactly what in the DOM looks like it might be staying in an odd state from the Chrome inspector.

Standard:
<img width="1022" alt="Screen Shot 2020-02-10 at 17 00 23" src="https://user-images.githubusercontent.com/7970809/74124670-2b2c9900-4c27-11ea-90c9-d4832fd06b2b.png">

After turning on the light:
<img width="1014" alt="Screen Shot 2020-02-10 at 17 03 32" src="https://user-images.githubusercontent.com/7970809/74124705-50210c00-4c27-11ea-963b-ed3b72bd6945.png">

**Comment by chaptergy** [comment]
Unfortunately I am unable to replicate your issue. You are using the chrome browser? And the cards where this issue appears are `entity-button` cards?
It's a really weird issue, since the theme only overwrites some css variables containing color values, accessing css states like `:active` shouldn't even be possible.

**Comment by teknetia** [comment]
Correct, I've tested in Chrome, Brave, Safari, the iOS App, and Fully Kiosk Browser on Android. I am also very confused about why it affects this theme but not others!

So I did some more messing about this morning and I think I might have found that there is another factor at play. HASS updated to 0.105 at some point during me testing things the last few days and since then I can replicate it on other themes now, including the default theme.

Now I am just more confused! But I guess this can at least be closed off. :P
